model/src/main/kotlin/config/RepositoryConfiguration.kt

```diff
@@ -48,19 +48,19 @@ data class RepositoryConfiguration(
 )
 
 @Suppress("EqualsWithHashCodeExist") // The class is not supposed to be used with hashing.
-class ExcludesFilter {
+internal class ExcludesFilter {
```

Comment:
Any particular reason to not make them even `private`? If so, would you mind adding that reason to the commit message?

Reply:
Good point, actually no. Update the change.